rules/php73/src/Rector/FuncCall/RemoveMissingCompactVariableRector.php

```diff
@@ -72,20 +72,53 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        $this->unsetUnusedArrayElements($node, $scope);
+        $this->unsetUnusedArguments($node, $scope);
+
+        if ($node->args === []) {
+            return new Array_();
+        }
+
+        return $node;
+    }
+
+    private function unsetUnusedArrayElements(Node $node, Scope $scope): void
+    {
         foreach ($node->args as $key => $arg) {
-            $argValue = $this->getValue($arg->value);
+            if (! $arg->value instanceof Array_) {
+                continue;
+            }
+
+            $array = &$arg->value;
```

Comment:
References are forbidden here. I need I add phpstan to cover it.

Could you refactor it?

Reply:
Sure